src/components/shared/Console.js

```diff
@@ -29,8 +29,8 @@ const OutputTest = styled.p`
   white-space: pre-line;
 `
 
-export default function Console({ executable, output }) {
-  const [showConsole, setShowConsole] = useState(false)
+export default function Console({ executable, output, show = false }) {
+  const [showConsole, setShowConsole] = useState(show)
```

Comment:
`DemoText` in this file needs `font` theme as well, the word `Details` is showing as Monaco on the Morello side

Reply:
thanks, has been addded would be. great if you could confirm. either way wont merge:)